fresque/__init__.py

```diff
@@ -10,15 +10,17 @@
 import logging.handlers
 import os
 import sys
-
 import flask
 import flask_fas_openid
-
+from inspect import getmembers, isfunction
 
 APP = flask.Flask(__name__)
+APP.jinja_env.trim_blocks = True
+APP.jinja_env.lstrip_blocks = True
```

Comment:
@pypingou our application will not get large number of requests for now and so I don't think we need this much of optimization on cost of readability.